assets/dsc/pwsh.dsc.resource.json

```diff
@@ -0,0 +1,239 @@
+{
+    "$schema": "https://aka.ms/dsc/schemas/v3/bundled/resource/manifest.json",
+    "type": "Microsoft/PowerShell",
+    "description": "DSC resource for PowerShell.",
+    "version": "0.1.0",
+    "tags": [
+        "PowerShell",
+        "DSC",
+        "Windows",
+        "Linux",
+        "macOS"
+    ],
+    "get": {
+        "executable": "pwsh",
+        "args": [
+            "-NoLogo",
+            "-NonInteractive",
+            "-NoProfile",
+            "-Command",
+            "./pwsh.resource.ps1 Get"
```

Comment:
What if we run the pwsh not from PSHome directory? I mean OS current directory != PSHome.

Maybe convert `pwsh.resource.ps1` to regular module and distribute like other in-box modules (Microsoft.PowerShell.Utility and etc.)?